.github/workflows/build_3.3.0.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Build and Test (3.3.0)"
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - '3.3.0/'
+      - '.github/workflows/main.yml'
+
+jobs:
+  run-build:
+    name: Run
+    secrets: inherit
+    uses: ./.github/workflows/main.yml
+    with:
+      spark: 3.3.0
+      scala: 2.12
+      java: 11
```

Comment:
will we have images for `java: 8` ?

Reply:
In the initial PR (this PR), there won't be. Main consideration as below:

1. Considering that https://hub.docker.com/r/apache/spark currently only has java 11.

2. The speed of DOI's PR review of new images will be relatively slow. Our top priority is to complete the review of the first image dockerfile. After this, update review will be very soon, only 2-3 days.

3. As planned, we will add some scripts to automatically generate dockerfiles of different versions in follow up(such as java/scala/spark version).

Reply:
got it, I ask it just because java 8 seems faster than 11/17 for now.

see https://github.com/apache/spark/pull/37799#discussion_r963181213